### PR TITLE
Degordian code style for matching TSLint

### PR DESCRIPTION
**Pull request by vilimco**
Code style for matching TSLint with some additional options for the same lint.
This is something that I use and have absolutely no problem with TSLint (i.e., I don't have to call `TSLint -> Fix current file`.

Some of the things it has covered (as far as I can remember now):
- proper ordering of fields and methods
- ending newlines
- remove trailing spaces
- end files with newline
- order imports properly

### CODE REVIEW

File: standards/typescript/tslint/Degordian code style.xml [hunk 1]
```diff
@@ -0,0 +1,240 @@
+<code_scheme name="Degordian code style" version="173">
+  <option name="LINE_SEPARATOR" value="&#xD;&#xA;" />
+  <PHPCodeStyleSettings>
```
Comment (krukru) [inline standards/typescript/tslint/Degordian code style.xml]: Can you remove php stuff from the config?

File: standards/typescript/tslint/Degordian code style.xml [hunk 2]
```diff
@@ -0,0 +1,235 @@
+<code_scheme name="Degordian code style" version="173">
+  <option name="LINE_SEPARATOR" value="&#xD;&#xA;" />
+  <TypeScriptCodeStyleSettings>
+    <option name="SPACE_BEFORE_FUNCTION_LEFT_PARENTH" value="false" />
+    <option name="ENFORCE_TRAILING_COMMA" value="WhenMultiline" />
+    <option name="SPACES_WITHIN_IMPORTS" value="true" />
+    <option name="IMPORT_SORT_MODULE_NAME" value="true" />
+    <option name="IMPORT_PREFER_ABSOLUTE_PATH" value="FALSE" />
+  </TypeScriptCodeStyleSettings>
+  <codeStyleSettings language="PHP">
```
Comment (krukru) [inline standards/typescript/tslint/Degordian code style.xml]: Sorry, this is also php :D

**Review by krukru — APPROVED** [review]
I like this, hopefully people will no longer have problems with tslint and realise that it is here for our benefit :)

**Comment by krukru** [comment]
EDIT: ignore this comment, wrong PR

Also please update npm version and hold off merging until I connect jenkins to publish the package

File: standards/typescript/tslint/Degordian code style.xml (hunk 2 above)
Comment (vilimco) [reply standards/typescript/tslint/Degordian code style.xml]: my bad, I updated storm and had to update some shortcuts to didn't update it till the end haha

**Comment by vilimco** [comment]
@krukru do we need to update npm version? We just added config file and readme? Does it need to be republished to npm? Nothing else changed. :)

**Review by leovujanic — APPROVED** [review]
👍

**Comment by bonzzy** [comment]
Where are with this? Can we merge with master and patch npm version?

**Review by bonzzy — APPROVED** [review]

**Comment by krukru** [comment]
Npm version is not needed since it is not an npm module

**Pull request merged**